src/Jira/Api.php

```diff
@@ -875,14 +882,23 @@ public function getProjectIssueTypes($project_key)
 	/**
 	 * Returns a list of all resolutions.
 	 *
-	 * @param string $project_key Project key.
-	 *
-	 * @return array|false
+	 * @return array
 	 * @since  2.0.0
 	 */
 	public function getResolutions()
 	{
-		return $this->api(self::REQUEST_GET, '/rest/api/2/resolution', array(), true);
+		if ( !count($this->resolutions) ) {
+			$resolutions = array();
+			$result = $this->api(self::REQUEST_GET, '/rest/api/2/resolution', array());
+
+			foreach ( $result->getResult() as $k => $v ) {
```

Comment:
Please do this:

1. remove `$k` variable, because array key isn't being used
2. rename `$v` variable into `$resolution_data`